src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -117,34 +106,37 @@ public String fillInstructorEmail(String value) {
     }
     
     public boolean clickEditInstructorLink(int instrNum) {
-        boolean isEditable;
-        if (instrNum == 1) {
-            editInstructorLink.click();
-            waitForElementVisibility(saveInstructorButton);
-            isEditable = editInstructorNameTextBox.isEnabled()
-                        && editInstructorEmailTextBox.isEnabled()
-                        && saveInstructorButton.isDisplayed();
-        } else {
-            String instructorNum = String.valueOf(instrNum);
-            WebElement editLink = browser.driver.findElement(By.id("instrEditLink" + instructorNum));
-            editLink.click();
-            
-            WebElement saveButton = browser.driver.findElement(By.id("btnSaveInstructor" + instructorNum));
-            waitForElementVisibility(saveButton);
-            
-            WebElement editInstructorNameTextBox = browser.driver.findElement(By.id("instructorname" + instructorNum));
-            WebElement editInstructorEmailTextBox = browser.driver.findElement(By.id("instructoremail" + instructorNum));
-            
-            isEditable = editInstructorNameTextBox.isEnabled()
-                        && editInstructorEmailTextBox.isEnabled()
-                        && saveButton.isDisplayed();
-        }
+        getEditInstructorLink(instrNum).click();
+        
+        WebElement saveButton = browser.driver.findElement(By.id("btnSaveInstructor" + instrNum));
```

Comment:
you might want to extract a method to get the save button, since such methods exist for most other elements on the page